dxp/demystifying_aui_input_tags.md

```diff
@@ -0,0 +1,727 @@
+| ------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `autoFocus`         | A `boolean` value that focuses the input on page load (e.g., `autoFocus="<%= true %>"`). This might not work with Singe Page Applications.                                                                                                                                             |
+| `bean`              | TODO: A bean to associate with the input component.                                                                                                                                                                                                                                    |
+| `cssClass`          | Adds classes to `<input class="form-control" />`                                                                                                                                                                                                                                       |
+| `data`              | A convenience attribute for pumping out data attributes from `key, value` pairs in a Java HashMap. For example, the `HashMap<"class-name", foo>` would render as the attribute `data-class-name = foo;`. You can also use the standard HTML data attribute `data-class-name="foo"`.    |
+| `dynamicAttributes` | `Map<String, Object>`                                                                                                                                                                                                                                                                  |
+| `disabled`          | A `boolean` value that enables or disables the `input` (e.g., `disabled="<%= true %>"`).                                                                                                                                                                                               |
+| `helpMessage`       | Adds an icon with tooltip next to label text that displays a tooltip with the `helpMessage`. This attribute accepts language keys.                                                                                                                                                     |
+| `id`                | Prepends the portlet namespace to the `id`, if no `id` is specified it's the same as the `name` attribute                                                                                                                                                                              |
```

Comment:
"," → ";"

And maybe end this with a period? I see you are doing that in most of the cells in this column, but not this one.